src/main.rs

```diff
@@ -9,47 +9,111 @@ extern crate serde_dynamodb;
 
 use rocket_contrib::json::Json;
 use rusoto_core::Region;
-use rusoto_dynamodb::{DynamoDb, DynamoDbClient, PutItemInput};
+use rusoto_dynamodb::{DynamoDb, DynamoDbClient, AttributeValue, GetItemInput, PutItemInput};
 use uuid::Uuid;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use serde_json::Value;
+
+thread_local!(static DDB: RefCell<DynamoDbClient> = RefCell::new(DynamoDbClient::new(Region::UsEast1)));
```

Comment:
fancyboi!

Reply:
haha, seemed like the best thing I could find. Wasn't sure how I'd pass around an ARC in the context of Rocket routes.